test/test_filters.rb

```diff
@@ -368,6 +368,24 @@ def select; end
         assert_equal "http://example.com/base", filter.absolute_url(page_url)
       end
 
+      should "not append a forward slash if input is '/'" do
+        page_url = "/"
+        filter = make_filter_mock({
+          "url"     => "http://example.com",
+          "baseurl" => "/base"
+        })
+        refute_equal "http://example.com/base//", filter.absolute_url(page_url)
```

Comment:
Can we `assert` that it is proper, rather than refuting this particular wrong case?